core/app/c/[communitySlug]/fields/FieldForm.tsx

```diff
@@ -61,21 +62,13 @@ type FormValues = z.infer<typeof schema>;
 const DEFAULT_VALUES = {
 	id: "",
 	name: "",
-	schemaName: null,
+	// can't be `null` and `isRelation: false` by default so should be undefined
+	schemaName: undefined,
 	slug: "",
 	isRelation: false,
-};
+} as const;
```

Comment:
this is an example of undefined behavior that `strict: true` now helped catch.

Because I needed to do `useForm<FormValues>` below, Typescript now yelled at me that you cannot set the default values as `schemaName: null` and `isRelation: false`.

The `as const` is necessary here to treat `isRelation` as `false`, rather than `boolean`.

Reply:
Scratch the above, i needed to revert that bc it would crash the app